statistics/cmsketch.go

```diff
@@ -162,8 +156,8 @@ func (c *CMSketch) updateTopNWithDelta(h1, h2 uint64, d []byte, delta uint64) bo
 		return false
 	}
 	for _, cnt := range c.topN[h1] {
-		if cnt.h2 == h2 && bytes.Equal(d, cnt.data) {
-			cnt.count += delta
+		if cnt.h2 == h2 && bytes.Equal(d, cnt.Data) {
```

Comment:
There are so many iterations like this to find a `TopNMeta`. Can we extract this by a function `func (c *CMSketch) findTopNMetaItem(h1, h2 uint64) *TopNMeta`